README.md

```diff
@@ -70,7 +66,7 @@ terraform {
   required_providers {
     databricks = {
       source = "databrickslabs/databricks"
-      version = "0.3.10"
+      version = "0.3.11"
```

Comment:
should be changed to 0.4.0?

Reply:
that's right before the release, i've forgotten to flip 0.3.10 with 0.3.11 so doing it now :D